src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,138 @@
 package co.inventorsoft.academy.collections.model;
 
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.Set;
+import java.util.*;
+import java.util.function.Function;
 
 public class Range<T> implements Set<T> {
 
+    List mylist = new ArrayList();
+
+    public static void main(String[] args) {
```

Comment:
Remove this method, remove commented code

Reply:
okey